src/components/Nav.jsx

```diff
@@ -1,12 +1,28 @@
-import './Nav.css';
+import styles from "./Nav.module.css";
+import Nav_logo from "../assets/nav_panda_logo_img.png";
+import Nav_user from "../assets/nav_user_img.png";
+import { Link } from "react-router-dom";
 
-const Nav = ({ leftChild, center, rightChild }) => {
+const Nav = () => {
   return (
-    <nav className="Nav">
-      <div className="nav_logo">{leftChild}</div>
-      <div className="nav_center">{center}</div>
-      <div className="nav_user">{rightChild}</div>
-    </nav>
+    <div className={styles.Nav}>
```

Comment:
클래스네임에는 절대 대문자는 지양해주세요!
```suggestion
    <div className={styles.nav}>
```